src/pages/spawn/spawnHost/SpawnHostCard.tsx

```diff
@@ -49,20 +50,30 @@ const spawnHostCardFieldMaps = {
     <>
       {host.volumes.map(({ id, displayName }) => (
         <div key={`volume_link_${id}`}>
-          <Link to={`${routes.spawnVolume}?volume=${id}`}>
+          <StyledRouterLink to={`${routes.spawnVolume}?volume=${id}`}>
             {displayName || id}
-          </Link>
+          </StyledRouterLink>
         </div>
       ))}
     </>
   ),
   "Home Volume": (host: MyHost) => (
     <span>
-      <Link to={`${routes.spawnVolume}?volume=${host?.homeVolumeID}`}>
+      <StyledRouterLink
+        to={`${routes.spawnVolume}?volume=${host?.homeVolumeID}`}
+      >
         {host?.homeVolumeID}
-      </Link>
+      </StyledRouterLink>
     </span>
   ),
+  IDE: (host: MyHost) =>
```

Comment:
If there is no IDE, will it just display` IDE:   `? Is it better if it displays nothing in that case?

Reply:
No it doesn't i added this bit so if the field is undefined it wont display anything 
https://github.com/evergreen-ci/spruce/pull/527/files#diff-5f8d2393c241c67cc24e5f5513d177dd0ffcae8470aba311c6770568a53ddaa3R23